daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -842,11 +887,17 @@ message CaseAlt {
 
     // name of the variant constructor
     // *Must be a valid identifier*
-    string variant = 2;
+    oneof variant {
```

Comment:
I suppose the reason why we can’t change this to a single `oneof interned_name` type and then change all field types to that is backwards compat?

Reply:
Correct.